src/main/java/ru/training/at/hw8/constants/SingleValueParameter.java

```diff
@@ -0,0 +1,19 @@
+package ru.training.at.hw8.constants;
+
+import ru.training.at.hw8.propertyloader.PropertyLoader;
+
+import java.util.Properties;
+
+import static ru.training.at.hw8.propertyloader.PropertyLoader.*;
+
+public class SingleValueParameter {
+    private static final Properties envProperties = getProperties();
+    public static final String KEY;
+    public static final String TOKEN;
+
+    static {
+        assert envProperties != null;
+        KEY = envProperties.getProperty("KEY");
+        TOKEN = envProperties.getProperty("TOKEN");
+    }
```

Comment:
You have got 2 methods for getting KEY and TOKEN:  public static String getKey() {
        return propertiesInstance.getProperty("KEY");
    }
    public static String getToken() {
        return propertiesInstance.getProperty("TOKEN");
    }
Why don't you use them here? If you don't need them - just delete

Reply:
Okay, thank you. Forgot to delete